recycle.xml

```diff
@@ -0,0 +1,210 @@
+<!-- 1 Enrichment 2 Reactors -->
```

Comment:
I'm fine with any approach as long as there is a recycle case regression test in 1.3